bitnami/flink/templates/taskmanager/deployment.yaml

```diff
@@ -0,0 +1,151 @@
+      {{- end }}
+      {{- if .Values.taskmanager.podSecurityContext.enabled }}
+      securityContext: {{- omit .Values.taskmanager.podSecurityContext "enabled" | toYaml | nindent 8 }}
+      {{- end }}
+      {{- if .Values.taskmanager.nodeSelector }}
+      nodeSelector: {{- include "common.tplvalues.render" (dict "value" .Values.taskmanager.nodeSelector "context" $) | nindent 8 }}
+      {{- end }}
+      {{- if .Values.taskmanager.tolerations }}
+      tolerations: {{- include "common.tplvalues.render" (dict "value" .Values.taskmanager.tolerations "context" $) | nindent 8 }}
+      {{- end }}
+      {{- if .Values.taskmanager.topologySpreadConstraints }}
+      topologySpreadConstraints: {{- include "common.tplvalues.render" (dict "value" .Values.taskmanager.topologySpreadConstraints "context" $) | nindent 8 }}
+      {{- end }}
+      initContainers:
+      {{- if .Values.taskmanager.initContainers }}
+      {{- include "common.tplvalues.render" (dict "value" .Values.taskmanager.initContainers "context" $) | nindent 8 }}
+      {{- end }}
+      containers:
+        - name: flink-taskmanager
+          image: {{ include "flink.image" . }}
+          imagePullPolicy: {{ .Values.image.pullPolicy | quote }}
+          {{- if .Values.taskmanager.containerSecurityContext.enabled }}
+          securityContext: {{- omit .Values.taskmanager.containerSecurityContext "enabled" | toYaml | nindent 12 }}
+          {{- end }}
+          {{- if .Values.diagnosticMode.enabled }}
+          command: {{- include "common.tplvalues.render" (dict "value" .Values.diagnosticMode.command "context" $) | nindent 12 }}
+          {{- else if .Values.taskmanager.command }}
+          command: {{- include "common.tplvalues.render" (dict "value" .Values.taskmanager.command "context" $) | nindent 12 }}
+          {{- end }}
+          {{- if .Values.diagnosticMode.enabled }}
+          args: {{- include "common.tplvalues.render" (dict "value" .Values.diagnosticMode.args "context" $) | nindent 12 }}
+          {{- else if .Values.taskmanager.args }}
+          args: {{- include "common.tplvalues.render" (dict "value" .Values.taskmanager.args "context" $) | nindent 12 }}
+          {{- end }}
+          {{- if .Values.taskmanager.lifecycleHooks }}
+          lifecycle: {{- include "common.tplvalues.render" (dict "value" .Values.taskmanager.lifecycleHooks "context" $) | nindent 12 }}
+          {{- end }}
+          ports:
+            - name: data
+              containerPort: {{ .Values.taskmanager.containerPorts.data }}
+            - name: rpc
+              containerPort: {{ .Values.taskmanager.containerPorts.rpc }}
+          env:
+            - name: FLINK_MODE
+              value: taskmanager
+            # Hosts and addresses are determined by the service full name, which maps to flink.*§.fullname
+            - name: FLINK_CFG_JOBMANAGER_RPC_ADDRESS
+              value: {{ include "flink.jobmanager.fullname" . }}
```

Comment:
Add a quote

Reply:
Essentially, we don't want that a fullnameOverride full of letters causes the deployment to fail

Reply:
This is still not done

Reply:
[a1ea30e](https://github.com/bitnami/charts/pull/14879/commits/a1ea30e138c58e236d89638236a630326b295fcc)